gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -93,8 +93,8 @@
  * </p>
  *
  * <p>
- *   This class runs in the {@link GobblinClusterManager}. The actual task execution happens in the in the
- *   {@link GobblinTaskRunner}.
+ *   The instance of this class is instantiated and runing along with the {@link GobblinHelixJobScheduler}.
```

Comment:
Rewording: This class is instantiated by the {@link GobblinHelixJobScheduler} on every job submission to launch the Gobblin job.